tests/transaction.ts

```diff
@@ -79,28 +82,18 @@ describe("Transaction", () => {
         },
       ],
     };
-    const requestSpy = jest.spyOn(axios, "request").mockResolvedValue({
-      status: 200,
-      data: sampleTransactionResponse,
-    });
+
+    nock(SERVER_BASE_URL, {
+      reqheaders: REQUEST_HEADERS,
+    }).post(`/partner/v3/orders/someOrderId/transactions`, _.matches(sampleCreateTransaction)).reply(200, sampleTransactionResponse);
+
     await expect(
       doshii.transaction.createTransaction(
         locationId,
         "someOrderId",
         sampleCreateTransaction
       )
     ).resolves.toMatchObject(sampleTransactionResponse);
-    expect(requestSpy).toBeCalledWith({
```

Comment:
i assume this assertion is removed because it has already been tested somewhere else?

Reply:
replaced all request spies with `nock`. This was the bulk of the work.